CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+This Code of Conduct applies to Code.gov, its repositories, communications, events and anyone participating in the Code.gov community [link to about page]. Your use of Code.gov means you accept and agree to abide by this Code of Conduct, which we may revise periodically. You should review this page from time to time. 
+
+## What we expect
+
+We expect all visitors and contributors to Code.gov to have a safe and engaging experience that is beneficial to our community. Members of our community have a diverse set of experiences and skillsets. As you engage our community, please ensure you are:  
+
+- __Considerate__: as an open source community, what you do will affect others. Ensure what you do is in the best interest of the project, and community at large. 
+- __Collaborative__: the essences of open source is collaborative in nature. Attempt to honor the spirit of open source by working together before conflict.
```

Comment:
There's some grammatical issues with this sentence.  Here's a potential revision for your review:
```
__Collaborative__: open source is collaborative in nature.  Attempt to honor the spirit of open source by working together and resolving issues before conflict arises.
```

Reply:
This sounds fine to me.